src/tribler-core/tribler_core/modules/metadata_store/tests/test_channel_metadata.py

```diff
@@ -331,7 +330,7 @@ def test_vsids(metadata_store):
 
     channel = metadata_store.ChannelMetadata.create_channel('test', 'test')
     metadata_store.vote_bump(channel.public_key, channel.id_, peer_key.pub().key_to_bin()[10:])
-    sleep(0.1)  # Necessary mostly on Windows, because of the lower timer resolution
+    freezer.move_to('2021-09-25')
```

Comment:
Cool feature!

Reply:
Yep! Wanted to use it for long time!

Reply:
Looks nice! Note that currently, freezegun does not mock asyncio's time, so if some async code inside a test calls things like `asyncio.sleep`, it can lead to [an unexpected behavior](https://togithub.com/spulec/freezegun/issues/290)